CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md

```diff
@@ -55,85 +54,71 @@ Images customization is done via overriding of [GiniImages.xcassets](https://git
-- Capture button
-  - Image &#8594; <span style="color:#009EDF">*cameraCaptureButton*</span> image asset
-- Import button
-	- Image &#8594; <span style="color:#009EDF">*documentImportButton*</span> image asset
-- Captured images stack indicator color &#8594; `GiniConfiguration.imagesStackIndicatorLabelTextcolor`
-- Flash toggle can be enabled through &#8594; `GiniConfiguration.flashToggleEnabled`
-- Flash button
-    - Image &#8594; <span style="color:#009EDF">*flashOn*</span> image asset
-    - Image &#8594; <span style="color:#009EDF">*flashOff*</span> image asset
-
-##### 4. QR code popup
-<br>
-<center><img src="img/Customization guide/QR code popup.jpg" height="70"/></center>
-</br>
-- Background color &#8594;  `GiniConfiguration.qrCodePopupBackgroundColor` using `GiniColor` with dark mode and light mode colors
-- Button color &#8594;  `GiniConfiguration.qrCodePopupButtonColor`
-- Text color &#8594;  `GiniConfiguration.qrCodePopupTextColor` using `GiniColor` with dark mode and light mode colors
-- Title &#8594; <span style="color:#009EDF">*ginicapture.camera.qrCodeDetectedPopup.buttonTitle*</span> localized string
-- Message &#8594; <span style="color:#009EDF">*ginicapture.camera.qrCodeDetectedPopup.message*</span> localized string
+### Single Page
+
+By default, the Gini Capture SDK is configured to capture single page documents.
+No further configuration is required for this.
+
+### Multi-Page
+
+The multi-page feature allows the SDK to capture documents with multiple pages.
+
+To enable this feature simply pass `true` to `GiniConfiguration.shared.multipageEnabled`.
+
+### Camera
+
+* Enable the flash toggle button:
+   To allow users toggle the camera flash pass `true` to `GiniConfiguration.shared.flashToggleEnabled`.
+
+* Turn off flash by default:
+   Flash is on by default, and you can turn it off by passing `false` to `GiniConfiguration.shared.flashOnByDefault`.
+
+ ### QR Code Scanning
+
+When a supported QR code is detected with valid payment data, the QR Code will be processed automatically without any further user interaction.
+The QR Code scanning maybe triggered directly without the need to analyze the document.
+
+If the QR code does not have a supported payment format then a popup informs the user that a QR code was detected, but it cannot be used.
+
+Please find more information in the [QR Code scanning guide](https://developer.gini.net/gini-mobile-ios/GiniCaptureSDK/qr-code-scanning-guide.html).
+
+### QR Code Only
+
+During QR Code only mode the capture and import controls will be hidden from the camera screen.
+
+For enabling QR code only mode the both flags `GiniConfiguration.shared.qrCodeScanningEnabled` and `GiniConfiguration.shared.onlyQRCodeScanningEnabled` should be `true`.
+
+More information about the customization is available [here](https://www.figma.com/file/H4WFJ4xaw4YNU4VaJYWiQq/iOS-Gini-Capture-SDK-2.0.0-UI-Customisation?node-id=243%3A3306&t=boRmrY5QfLYhGfPT-1)
```

Comment:
[Suggestion] I know it is just a link, but it could be misleading that we are releasing version 3.0 and Figma says it is 2.0

Reply:
I will be renamed soon, thank you 😉